praw/models/reddit/mixins/inboxtoggleable.py

```diff
@@ -6,7 +6,7 @@ class InboxToggleableMixin:
     """Interface for classes that can optionally receive inbox replies."""
 
     def disable_inbox_replies(self):
-        """Disable inbox replies for the item.
+        """Disable inbox replies for the item. This can only apply to a thing created by the user.
```

Comment:
```suggestion
        """Disable inbox replies for the item.
        
        .. note::
        
            This can only apply to items created by the authenticated user.
```

Let's add a note like this instead.